lib/utils/webpack.config.js

```diff
@@ -440,16 +441,35 @@ module.exports = (program, directory, suppliedStage, webpackPort = 1500, routes
     resolve: resolve(),
   })
 
-  if (modifyWebpackConfig) {
-    const modifiedWebpackConfig = modifyWebpackConfig(module(config), stage)
-    invariant(_.isObject(modifiedWebpackConfig),
-              `
-              You must return an object when modifying the Webpack config.
-              Returned: ${modifiedWebpackConfig}
-              stage: ${stage}
-              `)
-    return modifiedWebpackConfig
-  } else {
-    return module(config)
+  let exportedWebpackConfig = module(config)
+  if (!!modifyWebpackConfig) {
+    exportedWebpackConfig = modifyWebpackConfig(module(config), stage)
```

Comment:
If the user does not return a webpack configurator object here the subsequent calls to `exportedWebpackConfig.resolve()` will throw a `TypeError: Cannot read property 'resolve' of undefined`